docs/issues.md

```diff
@@ -120,3 +120,17 @@ Label changes probably requires some explanation:
 
 If you're trying to populate the `Labels` collection by hand, you might hit
 some exceptional behaviour due to these rules.
+
+### Locking and unlocking
+
+The easiest way to lock and issue is to pass details of the issue then use ```Lock()``` to lock it.
+
+```csharp
+var issue = client.Issue.Lock(owner, repo, issueNumber);
```

Comment:
I believe this has changed in a (very) recently merged PR. Could you take a look at the current docs to see if they need changing?

Reply:
It looks like this [PR](https://github.com/octokit/octokit.net/pull/2543) updated all of that.  The implementation is now:

`issuesClient.LockUnlock.Unlock()` and `issuesClient.LockUnlock.Lock()`

Reply:
@WillTheDeveloper I'd be glad to get this knocked out if you don't have the bandwidth to get them done - I'd need push access to your fork - let me know when you get the chance.  Thanks again for getting these changes made!